lib/hexdocs/bucket.ex

```diff
@@ -19,10 +19,47 @@ defmodule Hexdocs.Bucket do
     paths = MapSet.new(upload_files, &elem(&1, 0))
 
     upload_new_files(upload_files)
+
+    docs_config_js = build_docs_config_js(repository, package, version, all_versions)
+
+    [docs_config_js_upload_file] =
+      list_upload_files(repository, package, version, [docs_config_js], :unversioned)
+
+    upload_new_files([docs_config_js_upload_file])
```

Comment:
Include this file in `paths` above. Should be a little less duplicate code and we can parallelize the upload.